test_utils/mysql.go

```diff
@@ -37,6 +38,8 @@ const (
 	networkName = "moco-agent-test-net"
 )
 
+var MysqlSocketDir = path.Join(os.TempDir(), "moco-agent-test-mysql-socket-dir")
```

Comment:
The package "path" is for URL paths.
It no longer needs importing.

```suggestion
var MysqlSocketDir = filepath.Join(os.TempDir(), "moco-agent-test-mysql-socket-dir")
```